support/cas-server-support-pm/src/main/java/org/apereo/cas/pm/BasePasswordManagementService.java

```diff
@@ -119,8 +119,9 @@ public boolean change(final Credential c, final PasswordChangeBean bean) {
      * @param c    the credential
      * @param bean the bean
      * @return the boolean
+     * @throws InvalidPasswordException if new password fails downstream validation
      */
-    public boolean changeInternal(final Credential c, final PasswordChangeBean bean) {
+    public boolean changeInternal(final Credential c, final PasswordChangeBean bean) throws InvalidPasswordException {
```

Comment:
All great. Thank you. However, do unchecked exceptions need to be part of the throws clause? (I suppose they are, if you want to add javadocs; just checking)